### DISCUSSION TITLE
Issue with Movies.YML

### DISCUSSION THREAD

**Discussion by j-ktz** [meisnate12/Plex-Meta-Manager#74] [Q&A] [answered]
Any idea what's wrong here? Log says there is an issue on line 204 and line 7. Can I not add in a plex genre with trakt list?

<img width="740" alt="screenshot 2021-02-28 at 8 39 58 AM" src="https://user-images.githubusercontent.com/59346628/109420453-9535ee80-79a0-11eb-88e4-3a874f0acce9.png">
<img width="1200" alt="screenshot 2021-02-28 at 8 38 37 AM" src="https://user-images.githubusercontent.com/59346628/109420454-95ce8500-79a0-11eb-9edd-d494efc1e2ff.png">

**Comment by meisnate12** [comment]
Can you post the log and your metadata file?

**Comment by j-ktz** [comment]
Attached! 
[meta-10.log](https://github.com/meisnate12/Plex-Meta-Manager/files/6057001/meta-10.log)

https://pastebin.com/9785K2WE

**Comment by meisnate12** [comment, marked as answer]
the one at 204 is because your line at 203 has an indent when it shouldn't

I also fix the intention on your Reddit Top 250 collection which also caused me an error

**Comment by j-ktz** [comment]
@meisnate12 Sorry! I didn't know there were rules around indentation.
